subsys/fs/nvs/Kconfig

```diff
@@ -14,4 +14,26 @@ module = NVS
 module-str = nvs
 source "subsys/logging/Kconfig.template.log_config"
 
+config NVS_WRITE_CHECK
+	bool "Check written content"
+	help
+	  Check whether flash content match data written. Write API returns
+	  error if check fails as for other flash driver error.
+
+config NVS_WRITE_RETRY
+	bool "Retry write"
+	depends on NVS_WRITE_CHECK
+	help
+	  Retry write operation if flash content doesn't match to data written.
+	  This option can be enabled only on memories which allow writes to
+	  same location at las twice.
+
+config NVS_ALLOW_ATE_GAPS
+	bool "Allow big gaps in ATE area"
```

Comment:
Maybe change the title to "Scan presumed empty ATEs to recover last ate" or something like this, so that the title would more closely describe the purpose.